config/community-resources.json

```diff
@@ -0,0 +1,104 @@
+{
+  "data": [
```

Comment:
Thank you for the updates.  I don't think this needs `data`, it can just be the `array`

Reply:
Done !

Reply:
> Thank you for the updates. I don't think this needs `data`, it can just be the `array`

I have made the changes you asked and have the code formatted using prettier.